lib/features/sks-menu/presentation/sks_menu_screen.dart

```diff
@@ -63,11 +56,7 @@ class _SksMenuView extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     if (sksMenuData.meals.isEmpty) {
-      return const Scaffold(
-        body: Center(
-          child: CircularProgressIndicator(),
-        ),
-      );
+      return const SKSMenuLottieAnimation();
```

Comment:
Here we should check if menu's online or not and then show animation (list of meals now is only empty in case of error, if sks's closed cached data are provided, so this condition'll never work)